routes/general.js

```diff
@@ -53,6 +53,7 @@ router.get('/create', function (req, res) {
 });
 router.post('/create', (req, res) => {
   const snippet = new Snippet ({
+    //possibly using underscore and adding id here//
```

Comment:
What does this comment mean? is it a reminder, or did you forget to remove it?

Reply:
That was a reminder that I forgot to remove from the initial meeting.  I will remove it.